routes/posts.js

```diff
@@ -0,0 +1,49 @@
+import express from 'express'
+import * as postsModel from '../models/posts.js'
+
+export const postsRouter = express.Router()
+
+postsRouter.get('/', async function (req, res) {
+  if (req.query.week !== undefined) {
```

Comment:
Not a big fan of this way of handling the input, what if someone puts in the week and content parameters? They would not get what they expected here.